disorder-eithert/src/Disorder/Either.hs

```diff
@@ -0,0 +1,19 @@
+module Disorder.Either (
+    testEither
+  , testEitherT
+  ) where
+
+import           Control.Monad.Trans.Except (ExceptT, runExceptT)
+
+import           Data.Text (Text, unpack)
+
+import           Test.QuickCheck.Property
+
+
+testEither :: Testable a => (e -> Text) -> Either e a -> Property
+testEither t =
+  either (flip counterexample False . unpack . t) property 
```

Comment:
Not that it makes much difference - https://github.com/ambiata/disorder.hs/blob/master/disorder-core/src/Disorder/Core/Property.hs#L34